reanalysis/wrapper.py

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/env python3
+
+
+"""
+wrapper for reanalysis process
```

Comment:
Let's maybe not call this wrapper, but it's really the point where the pipeline is constructed, right?

I think we might want to reserve the wrapper for a non-CPG replacement of the analysis-runner, which sets up the right environment variables to launch this script.